### ISSUE TITLE
Error since release of babel-preset-babili 0.1.3

### ISSUE THREAD

**Issue by mistic** [babel/babili#583]
Hi guys

I'm using `babili-webpack-plugin` but since you release the version babel-preset-babili@0.1.3 2 days ago I have my production build with errors after minifying my javascript code. I haven't been able to identify the dependency with bug (so I dont know if it is related with this https://github.com/babel/babili/issues/577) however reverting babili to version `0.0.11` (which also reverts all the other babili packages for sure like `babel-plugin-minify-constant-folding`) everything starts working again.

This are the errors I have (I'm trying to link the source maps to identify the code before it was minified):

`Uncaught (in promise) TypeError: Cannot read property 'location' of undefined`

```javascript
function Router() {
    var _temp, _this, _ret;

    _classCallCheck(this, Router);

    for (var _len = arguments.length, args = Array(_len), _key = 0; _key < _len; _key++) {
      args[_key] = arguments[_key];
    }

    return _ret = (_temp = (_this = _possibleConstructorReturn(this, _React$Component.call.apply(_React$Component, [this].concat(args))), _this), _this.state = {
      match: _this.computeMatch(_this.props.history.location.pathname)
    }, _temp), _possibleConstructorReturn(_this, _ret);
  }
```

This code is from `react-router`.

**Comment by mistic** [comment]
Guys it's confirmed..
With those versions everything work:

```javascript
"babel-preset-babili": "0.1.2",

    "babel-plugin-minify-constant-folding": "0.1.1",
    "babel-plugin-minify-builtins": "0.1.1",
    "babel-plugin-minify-dead-code-elimination": "0.1.6",
    "babel-plugin-minify-flip-comparisons": "0.1.1",
    "babel-plugin-minify-guarded-expressions": "0.1.1",
    "babel-plugin-minify-infinity": "0.1.1",
    "babel-plugin-minify-mangle-names": "0.1.2",
    "babel-plugin-minify-numeric-literals": "0.1.1",
    "babel-plugin-minify-replace": "0.1.1",
    "babel-plugin-minify-simplify": "0.1.1",
    "babel-plugin-minify-type-constructors":"0.1.1",
    "babel-plugin-transform-inline-consecutive-adds":"0.1.1",
    "babel-plugin-transform-member-expression-literals":"6.8.3",
    "babel-plugin-transform-merge-sibling-variables":"6.8.4",
    "babel-plugin-transform-minify-booleans":"6.8.2",
    "babel-plugin-transform-property-literals":"6.8.3",
    "babel-plugin-transform-regexp-constructors":"0.1.1",
    "babel-plugin-transform-remove-console":"6.8.3",
    "babel-plugin-transform-remove-debugger":"6.8.3",
    "babel-plugin-transform-remove-undefined":"0.1.1",
    "babel-plugin-transform-simplify-comparison-operators":"6.8.3",
    "babel-plugin-transform-undefined-to-void":"6.8.2",
```

changing from `babel-plugin-minify-constant-folding@0.0.1` to `babel-plugin-minify-constant-folding@0.1.2` the build stop working again.

**Comment by boopathi** [comment]
I'm not sure what the bug is from the code your provided. Can you give me more details - like stack trace or minimal reproduction of the issue ?

**Comment by mistic** [comment]
@boopathi I can post the full stack trace of this particular issue I post but the code are the one I post above. As this is related with babel-plugin-minify-constant-folding and the code I post have an `Array.concat` do u believe that this issue is also related with the already reported code?

**Comment by jfurfaro** [comment]
I'm getting the same error, but it only occurs if the JS was compiled in my docker build. I tried fresh `npm install`'s on my mac, and everything compiled just fine. The minified file inside the docker image is actually different than my mac as well. Can't pinpoint the difference between environments

**Comment by dirtybit** [comment]
I think this is because of https://github.com/babel/babili/issues/577. Fixed in https://github.com/babel/babili/pull/578.

**Comment by boopathi** [comment]
As mentioned, `Array.prototype.concat` is fixed in #578. Will do a release soon.
